### ISSUE TITLE
Minor error in "Removed some shader duplication." commit

### ISSUE THREAD

**Issue by xvwyh** [Friendly0Fire/GW2Radial#95]
Hi. A rather belated report regarding https://github.com/Friendly0Fire/GW2Radial/commit/ab2f18c777aa2e2bd6085035c300ca3ac38166cf, as I just happened to update to the latest version and stopped using my fork. The commit has some minor mistakes in it resulting in washed-out colors. This line in particular: https://github.com/Friendly0Fire/GW2Radial/commit/ab2f18c777aa2e2bd6085035c300ca3ac38166cf#diff-ab8b23ed5836d16ed666687cc9608e46R240

```cpp
float3 fadedColor = lerp(color.rgb, luma, 0.33f);
float3 finalColor = fadedColor;
[...]
finalColor = lerp(fadedColor, finalColor, hoverFadeIn);
```
This code now makes no sense, because you're lerping from one value to the same value. The original code was lerping from `faded_color` to the original color: `g_vElementColor.rgb` in the case of mask images, and `color.rgb` in the case of alpha-blended full-color images (which, after the commit, are both essentially represented by `color.rgb`).

**Comment by Friendly0Fire** [comment]
Good catch, must've messed up while copy pasting!

**Comment by xvwyh** [comment]
But... but it should be `color.rgb`... Now you're lerping to a solid color all the time, even in case of fullcolor images.

**Comment by xvwyh** [comment]
Oh no... Look, I'm terribly sorry for bothering you about this the third time already, and I really should've just tested it myself (I didn't) before becoming confident that I found the cause of the issue... Turns out "comparing two pieces of code with your eyes" is not a very good way to fix bugs, who knew? Well, I tested it now, and this is the actual change that needed to be made in the first place. Sorry. Not saying that the rest are wrong and need to be reverted, just that this thing below was the actual culprit behind the discoloring.

```diff
 		color = g_vElementColor;
-		luma = dot(g_vElementColor.rgb, g_vLumaDot);
+		luma = dot(color.rgb, g_vLumaDot);
 	}
 	else
 	{
 		color = tex2D(texElementImageSampler, In.UV);
-		luma = dot(g_vElementColor.rgb, g_vLumaDot);
+		luma = dot(color.rgb, g_vLumaDot);
 	}
```
